src/server/routes/controllers/email.js

```diff
@@ -0,0 +1,119 @@
+const responseManager = require('./utilities/responseManager')
+const config = require('../../config.json')
+const nodemailer = require('nodemailer')
+const logger = require('../../logger')
+const crypto = require('crypto')
+const _ = require('lodash')
+
+const testAccount = {
+    email: process.env.EMAIL || config.email.email,
+    password: process.env.EMAIL_PASSWORD || config.email.password,
+    host: process.env.EMAIL_HOST || config.email.host
+}
+
+const SMTP_OPTIONS = { //TODO: Add SMTPS
+    host: `smtp.${testAccount.host}`,
+    port: 587,
+    secure: false,
+    auth: {
+        user: testAccount.email,
+        pass: testAccount.password
+    }
+}
+
+async function prepareTransporter(){
+    let transporter = nodemailer.createTransport(SMTP_OPTIONS)
+
+    await transporter.verify((err) => {
+        if(err) return console.log(err)
+        logger.info('Email server is ready to take our messages')
+    })
+
+    return transporter
+}
+
+function createToken(){
+    const token = crypto.randomBytes(20).toString('hex')
```

Comment:
Isn't it 40 in tests? Do they pass? If so, why?
Can we extract it to the config & env var?

Reply:
Thanks for reminding me of this, forgot to find out the reason for that strange behavior. I believe it's probably something simple to do with the behavior of buffers. I'll keep this updated.